mini_tutorials/openai_function_calling/example.py

```diff
@@ -72,7 +76,9 @@ def calculate(a: float, b: float, op: str) -> float:
     },
 }
 
-messages = [{"role": "user", "content": sys.argv[1]}]
+
+messages = [{"role": "user", "content": sys.argv[1]}},
+            {"role": "system", "content": f"You are an helpfful financial investor who overlooks the performace of stocks. Today is {date_today} ."}]
```

Comment:
* Typo `a helpful`
* Type `performance`

Also, Python will print out the date in the following way
```python
datetime.date(2023, 7, 7)
```

Which I am not sure if the LLM will understand. Can we maybe format it somehow and then tell the LLM what formatting we used?

For example

```python
date_today = datetime.date.today()
format="%m/%d/%Y"
print(f"Today is {date_today.strftime(format)} (format={format})")
```
```
Today is 07/07/2023 (format=%m/%d/%Y)
```

Reply:
Yes! 

` format="%Y/%m/%d"
date_today = date.today().strftime(format) `

and for the system prompt:

messages = [{"role": "user", "content": sys.argv[1]}},
            {"role": "system", "content": f"You are a helpful financial investor who overlooks the performance of stocks. Today is {date_today} . Note that the format of date is YYYY/MM/DD"}]

Reply:
@hrushik98 Can you please push a new commit on top of this branch with the above change? Or I can also do it, as you prefer:)

Reply:
Yes ,I have pushed the commit :)

https://github.com/jankrepl/mildlyoverfitted/pull/21/commits/617b7790e157e6ad5208b2bdff0960c292d6dd71